bittensor/commands/stake.py

```diff
@@ -573,6 +605,259 @@ def add_args(parser: argparse.ArgumentParser):
+                config.ss58 = str(hotkey_or_ss58)
+            else:
+                config.wallet.hotkey = str(hotkey_or_ss58)
+
+    @staticmethod
+    def add_args(parser: argparse.ArgumentParser):
+        set_childkey_take_parser = parser.add_parser(
+            "set_childkey_take", help="""Set childkey take."""
+        )
+        set_childkey_take_parser.add_argument(
+            "--netuid", dest="netuid", type=int, required=False
+        )
+        set_childkey_take_parser.add_argument(
+            "--hotkey", dest="hotkey", type=str, required=False
+        )
+        set_childkey_take_parser.add_argument(
+            "--take", dest="take", type=float, required=False
+        )
+        set_childkey_take_parser.add_argument(
+            "--wait_for_inclusion",
+            dest="wait_for_inclusion",
+            action="store_true",
+            default=True,
+            help="""Wait for the transaction to be included in a block.""",
+        )
+        set_childkey_take_parser.add_argument(
+            "--wait_for_finalization",
+            dest="wait_for_finalization",
+            action="store_true",
+            default=True,
+            help="""Wait for the transaction to be finalized.""",
+        )
+        set_childkey_take_parser.add_argument(
+            "--prompt",
+            dest="prompt",
+            action="store_true",
+            default=True,
+            help="""Prompt for confirmation before proceeding.""",
+        )
+        set_childkey_take_parser.add_argument(
+            "--y",
+            "--yes",
+            "--no_prompt",
+            dest="prompt",
+            action="store_false",
+            help="""Disable prompt for confirmation before proceeding. Defaults to Yes for all prompts.""",
+        )
+        bittensor.wallet.add_args(set_childkey_take_parser)
+        bittensor.subtensor.add_args(set_childkey_take_parser)
+
+
+class GetChildKeyTakeCommand:
+    """
+    Executes the ``get_childkey_take`` command to get your childkey take on a specified subnet on the Bittensor network to the caller.
+
+    This command is used to get your childkey take on a specified subnet on the Bittensor network.
+
+    Usage:
+        Users can get the amount or 'take' for their child hotkeys (``SS58`` address)
+
+    Example usage::
+
+        btcli stake get_childkey_take --hotkey <childkey> --netuid 1
+    """
+
+    @staticmethod
+    def run(cli: "bittensor.cli"):
+        """Get childkey take."""
+        try:
+            subtensor: "bittensor.subtensor" = bittensor.subtensor(
+                config=cli.config, log_verbose=False
+            )
+            GetChildKeyTakeCommand._run(cli, subtensor)
+        finally:
+            if "subtensor" in locals():
+                subtensor.close()
+                bittensor.logging.debug("closing subtensor connection")
+
+    @staticmethod
+    def _run(cli: "bittensor.cli", subtensor: "bittensor.subtensor"):
+        console = Console()
```

Comment:
should we use `bittensor.__console__` instead of `Console()` to be more consistent?

Reply:
For these commands, (that use this function) its mainly rich that gets used.